### ISSUE TITLE
`ValueError: time data '***221 Cashkonto - My Name' does not match format '%d.%m.%Y'`

### ISSUE THREAD

**Issue by hdarjus** [pspeter/statement-dl#4]
Hi, after executing `statement_dl flatex documents --all-files`, I get the following error:

```
Please login in the browser
Downloading files to /my/path/to/the/documents
More than 100 files, paging through results
Traceback (most recent call last):
  File "/Users/username/Library/Python/3.8/bin/statement_dl", line 8, in <module>
    sys.exit(main())
  File "/Users/username/Library/Python/3.8/lib/python/site-packages/statement_dl/__init__.py", line 92, in main
    args.func(args)
  File "/Users/username/Library/Python/3.8/lib/python/site-packages/statement_dl/flatex.py", line 42, in download_documents_from_args
    download_documents(
  File "/Users/username/Library/Python/3.8/lib/python/site-packages/statement_dl/flatex.py", line 93, in download_documents
    _download_pdfs(
  File "/Users/username/Library/Python/3.8/lib/python/site-packages/statement_dl/flatex.py", line 194, in _download_pdfs
    last_date = _parse_list_date(last_date_string)
  File "/Users/username/Library/Python/3.8/lib/python/site-packages/statement_dl/flatex.py", line 357, in _parse_list_date
    return datetime.strptime(date_string, "%d.%m.%Y").date()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '***221 Cashkonto - First Last Name' does not match format '%d.%m.%Y'
```

I am on a macbook, latest software. I installed Firefox now from its webpage, also geckodriver (via homebrew), and `statement_dl` via `pip3` today.

The error is with this line:
https://github.com/pspeter/statement-dl/blob/13e74c47b26a5cc4a6edd9f8e6f0d5fbcc028176/src/statement_dl/flatex.py#L357

My documents are called `***221 Cashkonto - ...` and not some datetime. I think that's the issue. Do you know what I could do to download all pdfs?

Thanks by the way for sharing your code!

Best

**Comment by pspeter** [comment]
Thanks for your report, hdarjus! :)

Looks like flatex might have changed their layout a bit. I don't have access to my PC this week to check it out, I'll try to fix it next week. But I'm also open to PRs if you want to try fix it yourself, it _probably_ needs just a small change in the last_date_string selector:

https://github.com/pspeter/statement-dl/blob/13e74c47b26a5cc4a6edd9f8e6f0d5fbcc028176/src/statement_dl/flatex.py#L191-L193

But as I said, I can't check it out this week.

**Comment by pspeter** [comment]
Hi, sorry I didn't get around to releasing a fix sooner, but PR #5 and release 2023.01.1 should have fixed this issue. Hope it works for you!
